### PR TITLE
Portal responsiveness

### PR DESCRIPTION
**Pull request by andychow326**
ref #2281

### CODE REVIEW

File: portal/src/graphql/portal/ManageLanguageWidget.module.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
 }
 
 .searchBox {
-  width: 300px;
+  max-width: 300px;
```
Comment (louischan-oursky) [inline portal/src/graphql/portal/ManageLanguageWidget.module.css:13]: The widget should have its label on the left, and its control on the right in the mobile view.

File: portal/src/graphql/adminapi/AuditLogScreen.tsx [hunk 2]
```diff
@@ -12,23 +12,20 @@ import {
 } from "react-router-dom";
 import {
   ICommandBarItemProps,
-  IDropdownOption,
   addDays,
```
Comment (louischan-oursky) [inline portal/src/graphql/adminapi/AuditLogScreen.tsx:15]: Investigate whether it is possible to let the list to take available height in mobile view. The pagination control should be visible without scrolling. Apply this to other list screen as well, such as the UsersScreen.

File: portal/src/graphql/portal/AppsScreen.module.css [hunk 3]
```diff
@@ -23,7 +23,7 @@
   display: block;
   padding: 20px;
   margin: 7px;
-  min-width: 300px;
+  width: 100%;
   max-width: 300px;
   min-height: 125px;
   border-radius: 3px;
```
Comment (louischan-oursky) [inline portal/src/graphql/portal/AppsScreen.module.css:29]: The horizontal margin of the apps card is not evenly distributed in mobile view.

File: portal/src/WizardContentLayout.tsx [hunk 4]
```diff
@@ -93,7 +94,7 @@ export default function WizardContentLayout(
   }, [makeGTMEventDataAttributes, trackSkipButtonEventData]);
 
   return (
-    <div className={styles.root}>
+    <div className={cn(styles.root, "mobile:grid-cols-6")}>
```
Comment (louischan-oursky) [inline portal/src/WizardContentLayout.tsx:97]: Why not col-span-full

File: portal/src/WizardContentLayout.tsx [hunk 5]
```diff
@@ -53,6 +53,7 @@ export function WizardDescription(
 ): React.ReactElement {
   return <Text block={true}>{props.children}</Text>;
 }
+import cn from "classnames";
```
Comment (louischan-oursky) [inline portal/src/WizardContentLayout.tsx:56]: This screen show the navigation menu in mobile view. But the navigation has no effect.

**Review by louischan-oursky — COMMENTED** [review]
Screenshot 2022-08-23 at 10 39 15 AM

1. The icon takes too much spacing. It should take constant space, like in desktop layout.
2. When we put the buttons in the 3rd row, it looks better if the 2nd row and the 3rd row is aligned to the left edge of the icon. What do you think @fungc-io @carmenlau

**Comment by fungc-io** [comment]
@louischan-oursky can i see a screenshot without the overlays

**Comment by andychow326** [comment]
@fungc-io 
image

**Comment by fungc-io** [comment]
Yes, the icons on the left should have constant width. ie the fingerprint icon should take up less horizontal space as the mail icon does.

**Comment by carmenlau** [comment]
How about we change the layout to 2 columns? The icon is column is fixed width. The buttons should be in the second column and the width of them should be the same.

File: portal/src/graphql/adminapi/AuditLogScreen.tsx (hunk 2 above)
Comment (louischan-oursky) [reply portal/src/graphql/adminapi/AuditLogScreen.tsx:15]: wontfix

**Comment by fungc-io** [comment]
@andychow326 @carmenlau @louischan-oursky let me quickly draw a mockup to illustrate what Carmen means there

**Comment by andychow326** [comment]
@fungc-io the new disign like this
image

**Comment by fungc-io** [comment]
How about: https://www.figma.com/file/msiE4O5imHONAG5EjhZeiZ/Authgear-UI?node-id=7297%3A109233

Frame 480

**Comment by andychow326** [comment]
@fungc-io But the `Social` label looks weird...
image

**Comment by fungc-io** [comment]
do this then

Frame 480-3

**Comment by andychow326** [comment]
> do this then
> 
> Frame 480-3

Done!

**Pull request merged**